presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixConfig.java

```diff
@@ -31,7 +31,7 @@ public void testDefaults()
         assertRecordedDefaults(recordDefaults(RubixConfig.class)
                 .setBookKeeperServerPort(CacheConfig.DEFAULT_BOOKKEEPER_SERVER_PORT)
                 .setDataTransferServerPort(CacheConfig.DEFAULT_DATA_TRANSFER_SERVER_PORT)
-                .setCacheLocation("/tmp")
+                .setCacheLocation(null)
```

Comment:
This looks strange. The field is `@NotNull`, but we're recording a null.

Reply:
This how you do a required config (something the user is forced to set).

Reply:
I know... it just looks weird and we should figure out a better way to represent that at some point.